drv/cosmo-hf/src/hf.rs

```diff
@@ -52,7 +68,15 @@ impl idl::InOrderHostFlashImpl for ServerImpl {
         if !matches!(protect, HfProtectMode::AllowModificationsToSector0) {
             return Err(HfError::Sector0IsReserved.into());
         }
-        todo!()
+        // Don't use the bulk erase command, because it will erase the entire
+        // chip.  Instead, use the sector erase to erase the currently-active
+        // virtual device.
+        for offset in 0..SLOT_SIZE_BYTES / SECTOR_SIZE_BYTES as u32 {
```

Comment:
**suggestion:** Would this be clearer / more straightfoward as `(0..SLOT_SIZE_BYTES).iter().step_by(SECTOR_SIZE_BYTES)`?